### ISSUE TITLE
ComboBox OnKeyDown bug!

### ISSUE THREAD

**Issue by rafaelrzacharias** [rds1983/Myra#357]
The bug is the following:

**Works:** If you left click with mouse on the dropdown it will expand the listbox within and show some items for you. Great.

**Doesn't work:** If you dont left click first, and just tab to get keyboard focus, then do up/down to select an item, all is well, the current item updates, but then if you left click to expand the listbox... it shows as empty.

**Expected result:** When I click on the dropdown, regardless if I had previously selected something with the keyboard or not, I would like to expand the combobox and see all items in the listbox within.
